jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -201,30 +201,29 @@ private Image pullBaseImage(
       ProgressEventDispatcher progressEventDispatcher)
       throws IOException, RegistryException, LayerPropertyNotFoundException,
           LayerCountMismatchException, BadContainerConfigurationFormatException {
+    EventHandlers eventHandlers = buildConfiguration.getEventHandlers();
     RegistryClient registryClient =
         buildConfiguration
             .newBaseImageRegistryClientFactory()
             .setAuthorization(registryAuthorization)
             .newRegistryClient();
 
-    ManifestTemplate manifestTemplate =
-        registryClient
-            .pullManifest(buildConfiguration.getBaseImageConfiguration().getImageTag())
-            .getManifest();
+    ManifestAndDigest manifestAndDigest =
+        registryClient.pullManifest(buildConfiguration.getBaseImageConfiguration().getImageTag());
+    ManifestTemplate manifestTemplate = manifestAndDigest.getManifest();
 
     // special handling if we happen upon a manifest list, redirect to a manifest and continue
     // handling it normally
     if (manifestTemplate instanceof V22ManifestListTemplate) {
-      buildConfiguration
-          .getEventHandlers()
-          .dispatch(
-              LogEvent.lifecycle(
-                  "The base image reference is manifest list, searching for linux/amd64"));
-      manifestTemplate =
+      eventHandlers.dispatch(
+          LogEvent.lifecycle(
+              "The base image reference is manifest list, searching for linux/amd64"));
+      manifestAndDigest =
           obtainPlatformSpecificImageManifest(
-                  registryClient, (V22ManifestListTemplate) manifestTemplate)
-              .getManifest();
+              registryClient, (V22ManifestListTemplate) manifestTemplate);
+      manifestTemplate = manifestAndDigest.getManifest();
```

Comment:
In the manifest list case, I have been thinking it makes sense to report the SHA of the manifest list. But now I think the digest of the actual image will be more helpful. Or, we could log both of them. @GoogleContainerTools/java-tools-build thoughts?

Reply:
The digest of the actual image will be more helpful. Logging both would be fine but i don't see any advantage in knowing the SHA of manifest list.

Reply:
Yeah, I think the image digest would be helpful.

That said, let's do this. We log the image digest as LIFECYCLE to display it to the user, and in addition to this, if the base image reference was a manifest list, log the manifest list digest as INFO. That will help the user and us easily determine if it was a manifest list by looking at a log when there's a problem. Makes sense?

Reply:
There is already a log as LIFECYCLE informing the user that the reference is manifest list. Should I follow it up with INFO log having the manifest list digest?

Reply:
Ah, we already inform that. Then I think this is already good enough.